sdcm/remote.py

```diff
@@ -271,6 +279,7 @@ def __init__(self, hostname, user="root", port=22, password="",
         self.extra_ssh_options = extra_ssh_options
         logger = logging.getLogger('avocado.test')
         self.log = SDCMAdapter(logger, extra={'prefix': str(self)})
+        self.log.info("{}, {}, {}, {}, {}, {}, {}".format(hostname, user, port, password, key_file, wait_key_installed, extra_ssh_options))
```

Comment:
looks like debug info: either remove or change the log level to debug and 
self.log.info("{hostname}, {user},.....".format(**locals())

Reply:
removed